src/instructlab/taxonomy/taxonomy.py

```diff
@@ -1,10 +1,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
+# Standard
+import logging
+
 # Third Party
 import click
 
 # First Party
 from instructlab import clickext
+from instructlab.configuration import storage_dirs_exist
+
+logger = logging.getLogger(__name__)
```

Comment:
nit: I see logger instance initializations in a few places in this PR but im not seeing them getting used anywhere 
dont know if we need them but not a blocker

Reply:
agh, apologies I was using the logger at first. I will re-push and rm